Index: java/org/apache/catalina/tribes/group/interceptors/MessageDispatch15Interceptor.java
===================================================================
--- java/org/apache/catalina/tribes/group/interceptors/MessageDispatch15Interceptor.java	(revision 1755541)
+++ java/org/apache/catalina/tribes/group/interceptors/MessageDispatch15Interceptor.java	(working copy)
@@ -18,7 +18,7 @@
 
 /**
  * @deprecated Originally provided an optional implementation that used Java 5+
- *             features. Now the minimum Java version is >=5, those features
+ *             features. Now the minimum Java version is &gt;=5, those features
  *             have been added to {@link MessageDispatchInterceptor} which
  *             should be used instead. This class will be removed in Tomcat
  *             8.5.x onwards.
