Index: java/org/apache/catalina/servlets/CGIServlet.java
===================================================================
--- java/org/apache/catalina/servlets/CGIServlet.java	(revision 1759564)
+++ java/org/apache/catalina/servlets/CGIServlet.java	(revision 1759565)
@@ -358,7 +358,7 @@
      *
      * @exception  IOException  if a write operation exception occurs
      *
-     * @deprecated Use {@link #printServletEnvironment(HttpServletRequest).
+     * @deprecated Use {@link #printServletEnvironment(HttpServletRequest)}.
      *             This will be removed in Tomcat 8.5.X onwards
      */
     @Deprecated
